packages/core/src/utils/breakpoints.ts

```diff
@@ -102,13 +102,22 @@ export const getValueForBreakpoint = (
   breakpoints: Breakpoint[],
   activeBreakpointIndex: number,
   fallbackBreakpointIndex: number,
-  variableName: string,
+  /** Provide the name for built-in styles to replace design tokens. Supported properties are:
+   * cfMargin, cfPadding, cfGap, cfWidth, cfHeight, cfBackgroundColor,
+   * cfBorder, cfBorderRadius, cfFontSize, cfLineHeight, cfLetterSpacing,
+   * cfTextColor, cfMaxWidth
```

Comment:
Not sure, if we should list all of these supported properties. Maybe we can point to `core/src/types.ts/StyleProps`

Reply:
That would be not correct as it's a type of all style props. This function is limited to a subset of it which is defined 20 lines above. Unfortunately, this list is not typed but just a `string[]` (probably because the type would not be used anywhere).

As it helped me to spot all the places using it the wrong way, I foudn it actually very helpful to show the list directly in the JSDoc as you will also see this when hovering over the function from another file when you want to use it.

Since the actual list and this comment are close to each other, I would prefer it this way if that's fine with you.